structure/extension/_ID.yaml

```diff
@@ -0,0 +1,24 @@
+%YAML 1.2
+---
+contact: https://github.com/glamberson/gedcom-evidence
+extension tags: _ID
+label: Evidence Identifier
+lang: en-US
+payload: '@<https://gedcom.io/terms/v7/record-INDI>@ | @<https://gedcom.io/terms/v7/record-FAM>@ | @<https://gedcom.io/terms/v7/record-SOUR>@'
```

Comment:
We don't currently have pointers that can point to multiple types of records in the spec, so this won't pass YAML validation. The workaround I know of now is replacing _ID with three different structures (perhaps _IDI, _IDF, and _IDS), each pointing to just one type of thing. Reverse pointers (from INDI to _ID) won't work because we also don't allow pointers to substructures.

As an aside, I have proposed before being loser with identifiers, but because many software implementations use relational databases having polymorphic pointers or non-record pointers can be problematic form implementers and don't seem to be likely to be added anytime soon.